toolkits/springer/packages/springer-box/scss/50-components/_box.scss

```diff
@@ -2,14 +2,20 @@
 	background-color: $box--background-color;
 	border-radius: $box--border-radius;
 	box-shadow: $box--box-shadow;
-	border-top: $box--border;
-	border-right: $box--border;
-	border-bottom: $box--border;
-	border-left: $box--border;
+	border: $box--border;
 	line-height: $box--line-height;
 	padding: $box--padding;
 }
 
+.c-box--background-grey {
+	background-color: $box--background-color-grey;
+}
+
+.c-box--borderless {
```

Comment:
I wonder if this is backwards and the modifiers should be `c-box--bordered` and `c-box-shadowed`

Reply:
I did have that thought too as it would be neater with less to override but then would it also be weird to have `c-box` not looking like a box..?

Reply:
Oh yeah, if the background is white then it's just nothing.
Makes sense. Although I'm not sure border and shadow are the same thing. Perhaps `c-box--shadow` could be split out?